test/test-format.ts

```diff
@@ -37,7 +37,7 @@ const OPTIONS: Options = {
   logger: {log: console.log, error: console.error, dir: nop}
 };
 
-test.serial('format should return false for well-formatted files', t => {
+test.serial('format should return true for well-formatted files', t => {
```

Comment:
Well that's confusing :) Was the title of the test just wrong before?

Reply:
true